config/machine.cfg

```diff
@@ -1,13 +1,18 @@
 [virtual_sdcard]
 path: ~/printer_data/gcodes
 on_error_gcode:
-   {% if printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
-      STATUS_LEDS COLOR="ERROR"
-   {% endif %}
-   {% if printer["gcode_macro _USER_VARIABLES"].probe_type_enabled == "dockable" %}
-       _PROBE_ON_ERROR_ACTION
-   {% endif %}
-    CANCEL_PRINT
+    {% if printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
+        STATUS_LEDS COLOR="ERROR"
+    {% endif %}
+    {% if printer["gcode_macro _USER_VARIABLES"].probe_type_enabled == "dockable" %}
```

Comment:
can you also add a reference to "dockable_virtual" as well ? I think it was forgotten and it's a good idea to add it as well.

Reply:
Yes ! this make sense.